warning related with closing commets

GCC says there is a warning in code related with `modules/posix/src/` C files 

```
./modules/posix/src/ciaaDevices.c:44:1: warning: "/*" within comment [-Wcomment]
 /*==================[inclusions]=============================================*/
 ^
```

--- a/modules/posix/src/ciaaDevices.c
+++ b/modules/posix/src/ciaaDevices.c
@@ -41,6 +41,7 @@
 /** \addtogroup CIAA_Firmware CIAA Firmware
  ** @{ */
 /** \addtogroup POSIX POSIX Implementation
+ ** @{ */
 /*==================[inclusions]=============================================*/
 #include "ciaaDevices.h"
 #include "ciaaPOSIX_stdlib.h"

--- a/modules/posix/src/ciaaMemory.c
+++ b/modules/posix/src/ciaaMemory.c
@@ -41,6 +41,7 @@
 /** \addtogroup CIAA_Firmware CIAA Firmware
  ** @{ */
 /** \addtogroup POSIX POSIX Implementation
+ ** @{ */
 /*==================[inclusions]=============================================*/
 #include "ciaaMemory.h"
 

--- a/modules/posix/src/ciaaPOSIX_stdio.c
+++ b/modules/posix/src/ciaaPOSIX_stdio.c
@@ -40,6 +40,7 @@
 /** \addtogroup CIAA_Firmware CIAA Firmware
  ** @{ */
 /** \addtogroup POSIX POSIX Implementation
+ ** @{ */
 /*==================[inclusions]=============================================*/
 #include "ciaak.h"            /* <= ciaa kernel header */
 #include "ciaaPlatforms.h"

